src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java

```diff
@@ -5,13 +5,14 @@
 import net.minecraft.world.entity.EquipmentSlot;
 import net.minecraft.world.item.ArmorMaterial;
 import net.minecraft.world.item.crafting.Ingredient;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Pretty much taken from minecraft's build-in armor materials
  */
 public enum ArmorMaterials implements ArmorMaterial {
-    MYRIAL_SUIT(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null);
-    
+    MYRIAL_EFFIGY(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null),
+    MYRIAL_SUIT(15, new int[] { 2, 5, 6, 2 }, 9, SoundEvents.ARMOR_EQUIP_IRON, 0f, 0f, null);
```

Comment:
Durability modifier and enchantability should be 0; this armor has custom durability and cannot be enchanted